vagrant.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Latest LTS as of Nov 22, 2016
+NODE_VERSION="6.9.1"
+
+# Install new nodejs.
+_node_filename="node-v${NODE_VERSION}-linux-x64.tar.xz"
+_node_url="https://nodejs.org/dist/v${NODE_VERSION}/${_node_filename}"
```

Comment:
I just tried `nvm` (because, unfortunately, even the nodejs in Ubuntu 16.04 is too old and has some features hidden behind the harmony flag), but found that it would actually compile node from source, which is quite slow. I guess I'll keep using the precompiled node.